client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ShareRedirectProps } from './types';
+
+export const ShareTemplate: React.ComponentType<ShareRedirectProps> = ({
+  redirectURL
+}) => {
+  const { t } = useTranslation();
+  return (
+    <a
+      title={t('buttons.share-on-twitter') || 'Share on Twitter'}
```

Comment:
I'm not completely opposed to using the `title` attribute here so that the browser will show this as a tooltip, but keyboard users won't see it. Do we really feel this is essential information? Is the word "Tweet" on the button not enough? If so, then perhaps we should figure out a way to give everyone this necessary information.

Reply:
Need your suggestions @ahmaxed @Sboonny 😅

Reply:
I think the word "tweet" is descriptive, we can delete the attribute and wait for feedback, if we need it to be more descriptive.

```suggestion
```